lib/fog/google/storage.rb

```diff
@@ -4,11 +4,18 @@
 module Fog
   module Storage
     class Google < Fog::Service
-      def self.new(options)
-        if options.keys.join(" ").include? "key"
-          Fog::Storage::GoogleJSON.new(options)
-        else
+      def self.new(options = {})
+        begin
+          fog_creds = Fog.credentials
+        rescue
```

Comment:
It'd be nice to rescue from a specific error, but we can save that for the next pass.